_component_design/tables-controls.md

```diff
@@ -0,0 +1,281 @@
+
+### Pagination
+{: .hxSubSectionTitle}
+
+The pagination controls group determines which page of a table is currently viewed.
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+{% figure [caption:"Filter regular / compact variations"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-pagination-1.svg){:width="100%"}
+{% endfigure %}
+
+{% endcolumn %}
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+{% figure [caption:"Filter buttons trancated / truncated-compact variations"] [class:"image bg-light border"] %}
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-pagination-2.svg){:width="100%"}
+{% endfigure %}
+
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+## Table controls - within
+{: .hxSectionTitle}
+The following controls may be found inside a table:
+- Select Icon
+- Action Cog Button
+- Sort table actions
+- Expand Table Row
+- Retract / Truncate table row
+ 
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Definition of within table actions and spacing specifications. NOT TO SCALE"] [class:"image bg-light border"] %}          
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-controls-within.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Actions cog
+{: .hxSubSectionTitle}
+
+The actions cog allows users to quickly initiate a number of create and edit workflows on an item in a table without drilling into an item’s detail page. These available actions are hidden behind the cog icon, allowing a user to interact with the cog to execute these actions. In obscuring these options behind a user click, the actions cog does not overload the users view of the table data.
+
+To ensure proper usage following these guidelines:
+
+- Only use a cog if you have three or more actions that may be performed on a table row.
+- Keep the actions list to seven or fewer actions.
+- Group like actions in the dropdown and avoid grouping unlike options.
+
+{% endcolumn %}
+
+{% column right:"hxCol-8 hxCol-xs-12 hxCol-sm-12 hxCol-md-8 hxCol-lg-8" %}
+{% figure [caption:"Actions cog triggered IxD. NOT TO SCALE"] [class:"image bg-light border"] %}          
+![]({{site.url}}/assets/images/components/tables/controls-tables/tables-actions-ixd.svg){:width="100%"}
+{% endfigure %}
+{% endcolumn %}
+
+</div>
+
+<div id="within" class="hxRow">
+
+{% column left:"hxCol-4 hxCol-xs-12 hxCol-sm-12 hxCol-md-4 hxCol-lg-4" %}
+
+### Variations and icon spacing
+{: .hxSubSectionTitle}
+
+
+To ensure proper usage following these guidelines:
+
+- If a checkbox is present it **will always** be the first option
```

Comment:
Comma needed between `present` and `it`; period at end.

Reply:
Also, change to `it is **always** the first option` for clarity.

Reply:
updated.